functions/screenshot.js

```diff
@@ -123,7 +124,13 @@ module.exports = async function screenshot({ page, context } = {}) {
     }
   }
 
-  const data = await page.screenshot(options);
```

Comment:
The feature works with the following tweaks:

``` javascript
  const data =
    selector !== null
      ? await (async() => {
          const elementHandle = await page.$(selector);
          return await elementHandle.screenshot(options);
        })()
      : await page.screenshot(options);
```

Reply:
Thanks @andyMrtnzP